cli/alterations/alteration.py

```diff
@@ -17,52 +17,54 @@
-    if "Encoding" in information:
-        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information["Encoding"])
+    if env.ALTERATION_ENCODING_TAG in information:
+        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information[env.ALTERATION_ENCODING_TAG])
     nifiConfiguration.create(nameCompose, merge)
 
 def createEncode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Encode_Decode.json",information)
-    if "Encoding" in information:
-        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information["Encoding"])
+    if env.ALTERATION_ENCODING_TAG in information:
+        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information[env.ALTERATION_ENCODING_TAG])
     nifiConfiguration.create(nameCompose, merge)
 
 
 
 def createAlteration(nifiConfiguration,allInformation):
-    name=allInformation["name"]
-    for alter in allInformation["alterations"]:
-        if alter["action"]=="Merge":
+    name=allInformation[env.NAME_TAG]
+    for alter in allInformation[env.ALTERATION_ALTERATIONS_TAG]:
+        if alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_MERGE_TAG:
             createMerge(nifiConfiguration,alter,name)
-        elif alter["action"]=="Encode":
+        elif alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_ENCODE_TAG:
             createEncode(nifiConfiguration,alter,name)
-        elif alter["action"]=="Decode":
+        elif alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_DECODE_TAG:
             createDecode(nifiConfiguration,alter,name)
     connectAlteration(nifiConfiguration,allInformation)
 
 
 def connectAlteration(nifiConfiguration,allInformation):
-    name=allInformation["name"]
-    for index,step in enumerate(allInformation["alterations"]):
+    name=allInformation[env.NAME_TAG]
+    for index,step in enumerate(allInformation[env.ALTERATION_ALTERATIONS_TAG]):
         if index == 0:
-            nifiConfiguration.makeConnection(name,nameActionReturn(step["action"],name))
+            nifiConfiguration.makeConnection(name,nameActionReturn(step[env.ALTERATION_ACTION_TAG],name))
         else:
-            nifiConfiguration.makeConnection(nameActionReturn(allInformation["alterations"][index-1]["action"],name),nameActionReturn(step["action"],name))
+            nifiConfiguration.makeConnection(nameActionReturn(allInformation[env.ALTERATION_ALTERATIONS_TAG][index-1][env.ALTERATION_ACTION_TAG],name),
+                                             nameActionReturn(step[env.ALTERATION_ACTION_TAG],
+                                                              name))
 
 def nameActionReturn(nameAction,nameSource):
```

Comment:
Please, follow the Flake8 guidelines:
- There should be a space before and after the =.
- There should be a space after each comma in the function call.